module/Swissbib/src/Swissbib/Controller/PersonDetailPageController.php

```diff
@@ -69,8 +69,8 @@ protected function addData(
         ViewModel &$viewModel, string $id, ElasticSearch $driver,
         array $bibliographicResources, array $subjectIds, array $subjects
     ) {
-        $media = $this->getMedia("Author", $driver);
-        $viewModel->setVariable("media", $media);
+        $media = $this->getMedias("Author", $driver);
```

Comment:
Changed the view model property name to reflect that there will be a list of medias instead of a single instance only

Reply:
Agreed